content/blog/deprecating-excalidraw-electron/index.md

```diff
@@ -0,0 +1,144 @@
+
+<figure>
+  <img src="excalidraw-pwa.png" alt="Excalidraw running in its own window.">
+  <figcaption>The Excalidraw PWA in a standalone window</figcaption>
+</figure>
+
+<figure>
+  <img src="excalidraw-icon.png" alt="Excalidraw icon on the macOS Dock.">
+  <figcaption>The Excalidraw icon on the macOS Dock</figcaption>
+</figure>
+
+### File system access
+
+Excalidraw makes use of [browser-nativefs](https://github.com/GoogleChromeLabs/browser-nativefs) for accessing the file system of the operating system. On supporting browsers, this allows for a true open-->edit-->save workflow and actual over-saving and "save as", with a transparent fallback for other browsers. You can learn more about this feature in my blog post [Reading and writing files and directories with the browser-nativefs library](browser-nativefs/).
+
+### Drag and drop support
+
+Files can be dragged and dropped onto the Excalidraw window just as in native applications. On a browser that supports the [File System Access API](https://web.dev/file-system-access/), a dropped file can be immediately edited and the modifications be saved in the original file. This is so intuitive that you sometimes forget that you are dealing with a Web app.
+
+### Clipboard access
+
+Excalidraw works well with the operating system's clipboard. Entire Excalidraw drawings or also just individual objects can be copied and pasted in `image/png` and `image/svg+xml` format, allowing for an easy integration with other native tools like [Inkspace](https://inkscape.org/) or Web-based tools like [SVGOMG](https://jakearchibald.github.io/svgomg/).
+
+<figure>
+  <img src="clipboard.png" alt="Excalidraw context menu showing the 'copy to clipboard as SVG' and 'copy to clipboard as PNG' menu items.">
+  <figcaption>The Excalidraw context menu offering clipboard actions</figcaption>
+</figure>
+
+### File handling
+
+Excalidraw already supports the experimental [File Handling API](https://web.dev/file-handling/), which means `.excalidraw` files can be double-clicked in the operating system's file manager and open directly in the Excalidraw app, since Excalidraw registers as a file handler for `.excalidraw` files in the operating system.
+
+### Declarative link capturing
+
+Excalidraw drawings can be shared by link—here is an [example](https://excalidraw.com/#json=4646308765761536,jwZJW8JsOM75vdhqG2nBgA). In the future, if people have Excalidraw installed as a PWA, such links will not open in a browser tab, but launch a new standalone window. This will work thanks to [declarative link capturing](https://github.com/WICG/sw-launch/blob/master/declarative_link_capturing.md), an, at the time of writing, bleeding-edge proposal for a new Web platform feature.
```

Comment:
```suggestion
Excalidraw drawings can be shared by link—here is an [example](https://excalidraw.com/#json=4646308765761536,jwZJW8JsOM75vdhqG2nBgA). In the future, if people have Excalidraw installed as a PWA, such links will not open in a browser tab, but launch a new standalone window. This will work thanks to [declarative link capturing](https://github.com/WICG/sw-launch/blob/master/declarative_link_capturing.md), a, at the time of writing, bleeding-edge proposal for a new Web platform feature.
```

Reply:
Not in this case. It's "an". Other [example](https://www.usenix.org/publications/login/fall2020/geer#node-259478:~:text=an%20%E2%80%9Cat%20the%20time%20of%20writing%E2%80%9D).